### PR TITLE
taxonomy: Add even more synonyms to `jp_allergens_1`

### PR DESCRIPTION
**Pull request by Naruyoko**
### What

Extends #9502, referring to [the same document](https://www.caa.go.jp/policies/policy/food_labeling/food_sanitation/allergy/assets/food_labeling_cms204_210514_01.pdf).

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -91019,6 +91020,7 @@ es:Caballa
 fi:makrillit
 fr:Maquereaux
 it:Sgombro
+ja:サケ, 鮭, しゃけ, シャケ
```
Comment (benbenben2) [inline taxonomies/categories.txt:91023]:
```suggestion
ja:鮭, しゃけ, シャケ
```

Comment (benbenben2) [reply taxonomies/categories.txt:91023]: サケ appears 2 times in the same file, for different categories. This is the reason why the test fails. 
After looking at wikipedia page and translation, I suggest to remove it  from mackerels and keep it for salmons.
What do you think?

Comment (Naruyoko) [reply taxonomies/categories.txt:91023]: Right, I should have moved サバ etc. there instead, but probably copied the wrong parts of the line. Actually, the correct translations for mackerels were already there, but for some reason assigned to `en:Mackerel fillets`.

Comment (Naruyoko) [reply taxonomies/categories.txt:91023]: I pushed a fix.

**Comment by Naruyoko** [comment]
It looks like there are still conflicts in the taxonomy. Some of them seem to be unrelated to my changes, but others are relevant here, for example ビーフ. I fixed some of the easy ones (and hopefully all the ones that I broke here), preferring to designate to the parent entries.

**Comment by benbenben2** [comment]
> It looks like there are still conflicts in the taxonomy. Some of them seem to be unrelated to my changes, but others are relevant here, for example ビーフ. I fixed some of the easy ones (and hopefully all the ones that I broke here), preferring to designate to the parent entries.

Try to click on "update branch" in your PR. That should merge with new PR that have been merged. And hopefully remove the failed test and the POD error.

**Comment by Naruyoko** [comment]
> > It looks like there are still conflicts in the taxonomy. Some of them seem to be unrelated to my changes, but others are relevant here, for example ビーフ. I fixed some of the easy ones (and hopefully all the ones that I broke here), preferring to designate to the parent entries.
> 
> Try to click on "update branch" in your PR. That should merge with new PR that have been merged. And hopefully remove the failed test and the POD error.

I don't see the button, which I think is because I'm trying to merge into `jp_allergens_1` rather than `main`. Should I merge `main` into mine instead?

**Comment by benbenben2** [comment]
> > > It looks like there are still conflicts in the taxonomy. Some of them seem to be unrelated to my changes, but others are relevant here, for example ビーフ. I fixed some of the easy ones (and hopefully all the ones that I broke here), preferring to designate to the parent entries.
> > 
> > 
> > Try to click on "update branch" in your PR. That should merge with new PR that have been merged. And hopefully remove the failed test and the POD error.
> 
> I don't see the button, which I think is because I'm trying to merge into `jp_allergens_1` rather than `main`. Should I merge `main` into mine instead?

I just clicked on the button for the pr `jp_allergens_1` to `main`. Let's see if it changes something. Otherwise, you can try to pr directly to main as you suggested.

**Comment by benbenben2** [comment]
@Naruyoko, maybe that could be easier to make your pr to main directly instead of my branch.
At your place I would try to rollback to previous commit where there was only your changes and apply those changes in a new branch based on main branch.

**Comment by Naruyoko** [comment]
> @Naruyoko, maybe that could be easier to make your pr to main directly instead of my branch. At your place I would try to rollback to previous commit where there was only your changes and apply those changes in a new branch based on main branch.

Okay. I made a new PR.

**Comment by Naruyoko** [comment]
Replaced by #9594.

**Pull request closed without merging**